crates/typst-library/src/layout/container.rs

```diff
@@ -201,8 +191,7 @@ pub enum InlineItem {
 /// )
 /// ```
 ///
-/// Blocks are also useful to force elements that would otherwise be inline to
-/// become block-level, especially when writing show rules.
+/// ブロックは、特に show ルールを記述する際、本来インラインとなる要素を強制的にブロックレベルとして扱う場合にも有用です。
```

Comment:
```suggestion
/// ブロックは、特にshowルールを記述する際、本来インラインとなる要素を強制的にブロックレベルとして扱う場合にも有用です。
```
[lint](https://github.com/typst-jp/typst-jp.github.io/pull/83)が通らずエラーが起きてしまっているため、半角文字と全角文字の間のスペースは除去してください。

Reply:
補足すると、これは`mise run textlint-html`で確認できます。

Reply:
お二人ともありがとうございます。
textlintに関しては次から気をつけます。
`mise run textlint-html` (と `mise run textlint-md` )は`CONTRIBUTING.md`に書いておいてくださると今後参加される方が助かると思います(devcontainerのところで言及されていますが、devcontainerを使っていないのでスルーしていました)。

---

borderの訳語は「枠線」に統一しました。

Reply:
ご指摘ありがとうございます！
- #181 で対応いたします。

Reply:
また補足となりますが、https://github.com/typst-jp/typst-jp.github.io/blob/main/CONTRIBUTING.md の最後の文にありますようにmise環境でもVS Codeのtasksは使用できるため、個人的には`> Tasks: Run task`を`Ctrl + Shift + T`に割り当てて利用しています。
> `> Tasks: Run task`はDocker環境でなく、miseで環境構築した際にも使用できます。